src/lvb.py

```diff
@@ -0,0 +1,21 @@
+# Written by David Bunn, 2024 (modified from mpboot.py)
```

Comment:
Remove this and add the Apache 2 licence at the top of every new file. This can be found at the top of all the other files. You'll also need to add yourself to the developers within 'Print.py'.